packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -120,43 +154,178 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
 
   return (
     <Box
-      className={cx(
-        'iui-expandable-block',
-        {
-          'iui-with-caption': !!caption,
-          'iui-expanded': expanded,
-          'iui-small': size === 'small',
-          'iui-borderless': styleType === 'borderless',
-        },
-        className,
+      className={cx('iui-expandable-header', className)}
+      role='button'
```

Comment:
this might be a good time to use a button instead of a div. See https://github.com/iTwin/iTwinUI/pull/1321#discussion_r1213349992

Reply:
I updated this to a button and removed the `role='button'`, assuming it's not needed anymore

Reply:
yes, this is good. Some additional notes:
1. it should be possible for it to be both expanded and disabled at the same time.
2. tabindex is not needed because `<button>`s are already focusable.
3. onKeyDown is not needed because it is already handled when using `<button>` with `onClick`.
4. user's `onClick` prop needs to be combined with your handler.
5. add `type='button'` to prevent accidental form submissions

```jsx
  const { className, children, label, onClick: onClickProp, ...rest } = props;
  const { isExpanded, setExpanded, disabled, onToggle } = useSafeContext(
    ExpandableBlockContext,
  );

  return (
    <Box
      as='button'
      type='button'
      className={cx('iui-expandable-header', className)}
      aria-expanded={isExpanded}
      aria-disabled={disabled}
      onClick={mergeEventHandlers(onClickProp, () => {
        if (disabled) {
          return;
        }
        setExpanded(!isExpanded);
        onToggle?.(!isExpanded);
      })}
      ref={forwardedRef}
      {...rest}
    >
      {children ?? (
        <>
          <ExpandableBlock.ExpandIcon />
          <ExpandableBlock.LabelArea>
            <ExpandableBlock.Title>{label}</ExpandableBlock.Title>
          </ExpandableBlock.LabelArea>
        </>
      )}
    </Box>
  );
```

after all this, test again in NVDA to confirm behavior is unchanged.

Reply:
Thanks for the suggestion. I added these changes and re-tested behavior + NVDA, working as expected still

Reply:
tabindex needs to be removed

Reply:
removed

Reply:
I am having trouble passing the `should trigger onToggle when clicked with Enter or Spacebar` unit test now because we eliminated the keyDown handler. I tried to add it back in this code but `mergeEventHandlers` would only accept MouseEvent argument and not KeyBoard, what do? It's worth noting that I have manually tested keyboard events (space, enter) on the header with a custom `onToggle` function and can verify that `onToggle` does get triggered, so I'm not sure why this test doesn't pass :-s